files/chef-marketplace-cookbooks/chef-marketplace/templates/default/chef-server.rb.erb

```diff
@@ -13,8 +13,10 @@ data_collector["token"]                     = "<%= node["chef-marketplace"]["aut
 nginx["non_ssl_port"]                       = false
 nginx["ssl_port"]                           = 8443
 nginx["dhparam_key_length"]                 = 1024
+nginx["ssl_protocols"]                      = "TLSv1.2"
```

Comment:
Probably could use a comment here explaining that this is cool cuz the Chef Server nginx only talks to automate's nginx.